docs/installation.md

```diff
@@ -26,13 +24,39 @@ git clone https://github.com/trilitech/jstz.git
 Both `jstz` and Octez are packaged with Nix, a package manager and system configuration tool that makes building from sources easy! See the [Nix docs](https://nixos.org/download.html) for instructions for your system. Additionally, ensure [Nix flakes are enabled](https://nixos.wiki/wiki/Flakes#Enable_flakes).
 :::
 
-### Clang and llvm 📜
+#### LLVM 🛠️
+
```

Comment:
ìs it better to have a copy of the installation guide? I was looking for modularised section but I suppose can't do it in md